h2/utilities.py

```diff
@@ -330,11 +330,9 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
         )
 
     # If ':status' pseudo-header is not there in a response header, reject it
-    if (
-        hdr_validation_flags.is_response_header and
-        b':status' not in seen_pseudo_header_fields and
-        u':status' not in seen_pseudo_header_fields
-    ):
+    seen_status_field = (b':status' in seen_pseudo_header_fields or
```

Comment:
@sunu I believe that the parens on their own lines is actually a style thing for this project that @Lukasa prefers.